cloudbuild.yaml

```diff
@@ -27,12 +27,27 @@ steps:
       - '-c'
       - 'date +%Y%m%d%H%M%s > .tag'
 
+  - id: 'Decrypt secrets required for build'
+    name: 'gcr.io/cloud-builders/gcloud'
+    entrypoint: 'sh'
+    args:
+      - '-c'
+      - |
+        _VALUES_PATH=./devops/secrets
+        gcloud kms decrypt \
+          --ciphertext-file=$${_VALUES_PATH}/secrets-$(cat .network).enc \
+          --plaintext-file=$(cat .network).env \
+          --location=global \
+          --keyring=origin \
+          --key=cloudbuild
+
   - id: 'Build docker image'
     name: 'gcr.io/cloud-builders/docker'
     entrypoint: '/bin/bash'
     args:
       - '-c'
       - |
+        source $(cat .network).env
```

Comment:
Personally would prefer if we didn't stuff a whole ton of custom build config into a .env(encrypted or not).  I liked @franckc's idea for using trigger substitutions for build args.